public/components/visualizations/charts/lines/line.tsx

```diff
@@ -36,7 +38,7 @@ export const Line = ({ visualizations, layout, config }: any) => {
   const {
     data: {
       explorer: {
-        explorerData: { jsonData },
+        explorerData: { schema, jsonData, datarows },
```

Comment:
datarows isn't used anywhere so should remove this as well

Reply:
Ok